doc/configuration.markdown

```diff
@@ -124,4 +124,13 @@ which will act as the cache key generator:
 	end
 	
 For more options see the [Rack::Request documentation](http://rack.rubyforge.org/doc/classes/Rack/Request.html)
-	
+
+### `use_native_ttl`
+
+Passes on the expiration timestamp to the cache store. This may be necessary
```

Comment:
something like `that support it, like memcached and redis` could be good to not make ppl think heap/file support it ...